_docs/_latest/develop/dev/testing.md

```diff
@@ -59,3 +59,18 @@ If you want to execute a subset of tests (e.g., all tests in a specific subdirec
     ```bash
     docker -l error exec -it build /bin/bash -c "python -m unittest discover -s tests/common -p '*_test.py'"
     ```
+
+### Continual Testing
```

Comment:
qq: I do not use docker for my testings.  So, once the tests are run in docker, is the docker container left running and not teared down?  If that's the case, we should document how to tear down.